.github/workflows/codeql.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: codeQL
+
+on:
+  push:
+    branches: [development/*]
+  pull_request:
+    branches: [development/*]
```

Comment:
we should be adding those branches:
- development/*
- stabilization/*
- hotfix/*

Reply:
Done